backend/src/main/kotlin/com/infowings/catalog/data/guid/GuidService.kt

```diff
@@ -0,0 +1,101 @@
+package com.infowings.catalog.data.guid
+
+import com.infowings.catalog.common.*
+import com.infowings.catalog.common.objekt.PropertyUpdateResponse
+import com.infowings.catalog.common.objekt.Reference
+import com.infowings.catalog.data.Subject
+import com.infowings.catalog.data.aspect.toAspectPropertyVertex
+import com.infowings.catalog.data.aspect.toAspectVertex
+import com.infowings.catalog.data.objekt.*
+import com.infowings.catalog.data.reference.book.toReferenceBookItemVertex
+import com.infowings.catalog.data.subject.toSubject
+import com.infowings.catalog.data.subject.toSubjectVertex
+import com.infowings.catalog.storage.*
+import com.orientechnologies.orient.core.record.ODirection
+
+data class EntityMetadata(val guid: String, val entityClass: String, val id: String)
```

Comment:
As long as it is transfered through the wire as part of API, it would be better to add this class to `common` project and make it `@Serializable`. It is also makes sense to enumerate entity classes and return entity type as Enum, that would also comfortably reside in `common` project.

Reply:
There is already OrientClass enum class. I'll move it to common

Reply:
After rethinking on this I don't think it is a good idea to keep OrientClass enum somewhere in common domain. Just because all details about orient must be kept as inside as possible.

However it could make sense to introduce enum type like EntityClass assuming it is enum about classes of business entities having nothing to do with Orient.